task_geo/data_sources/noaa/references.py

```diff
@@ -109,7 +120,7 @@ def load_dataset(dataset_name):
 
     Returns:
         pandas.DataFrame
-    '''
+    """
 
     dataset_params = DATASETS.get(dataset_name)
 
```

Comment:
You should create the folder "data" if it doesn't exist yet. I get an error running this for now.

Reply:
Well, it doesn't preserve indentation, but you know how to do it better than me :)